sql/CMakeLists.txt

```diff
@@ -21,35 +21,35 @@ set(IMMUTABLE_API_SOURCE_FILES
 
 # The rest of the source files defining mostly functions
 set(SOURCE_FILES
-  pre_install/types.functions.sql
-  hypertable.sql
+  bgw_scheduler.sql
```

Comment:
These changes seem unrelated to PR.

Reply:
I sorted the files to make is easier to find duplicates. Unsorted it again.